popde/density_estimate.py

```diff
@@ -232,6 +243,7 @@ def evaluate_with_transf(self, points):
             std_points = transf_points
 
         # Rescaling
+        #Transform data x' = x / c to shrinks or stretches the data
```

Comment:
it's not clear whether the rescaling factor (values of self.rescale) are notated as 'c' here or as '1/c'. Clarify please and check in transform_utils.py.

Reply:
fixed.